README.md

```diff
@@ -112,3 +113,29 @@ unserializable string.
 
 Exceptions will also be serialized. Stack Traces, filenames, line numbers and previous exceptions will be omitted
 when `kernel.debug` is set to `false` (= in productive environments) to avoid leaking potentially sensitive information.
+
+### Serialized Protobuf view
+If you wish to return Protobuf.
+
+```yaml
+//fos_rest.yaml
+fos_rest:
+    view:
+        mime_types:
+            proto: ['application/x-protobuf'] # You can specify whatever mime type you want, just map it to "protobuf".
```

Comment:
The comment is slightly off. The mapping should be "proto" and not "protobuf" as stated in the comment